src/docs/ChannelStateContext.md

```diff
@@ -24,6 +24,8 @@ The ChannelStateContext provides the following properties:
 
 -  **mutes** Muted users
 
+- **notifications** Channel notifications 
```

Comment:
this can be more descriptive, maybe "Temporary notifications added to the message list on specific user/message actions"